cmd/swarm/flags.go

```diff
@@ -65,6 +65,16 @@ var (
 		Usage:  "URL of the Ethereum API provider to use to settle SWAP payments",
 		EnvVar: SwarmEnvSwapBackendURL,
 	}
+	SwarmSwapPaymentThresholdFlag = cli.Uint64Flag{
+		Name:   "swap-payment-threshold",
+		Usage:  "honey amount indebted to a peer at which you will initiate payment",
```

Comment:
This is only from one peer's point of view. I would change this to "honey amount at which a payment is triggered"

Reply:
I think in anything outside of the test directory, we should make the comments and variable names as if it is from the perspective of the node operator.

Reply:
Maybe, but even then, this node operator can cross the payment threshold in both ways. The threshold doesn't define if you are indebted or not, which is defined only in which direction you cross it.

I would like to insist on this being changed.

Reply:
Actually, this threshold *does* define whether a node is indebted or not, because the threshold which is set here only applies to what this particular node does. Sure, there are other payment thresholds in the network, and the node can cross these from the other direction (and get paid). but this particular threshold is only about how much the node that the user is about to boot up must be indebted to other peers until payment will be initiated.

Reply:
What if we would add a log Warn on the creditor node that a peer crossed the threshold and is indebted with him?

Check this exact comment I made on another PR: https://github.com/ethersphere/swarm/pull/1754#discussion_r324301619

If we did this (and if we add some other metric or anything to help nodes track that peers got indebted to them), then your comment would not apply.